api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2304,15 +2304,15 @@ public void removeObject(String bucketName, String objectName)
    * objectNames.add("my-objectname1");
    * objectNames.add("my-objectname2");
    * objectNames.add("my-objectname3");
-   * for (Result<DeleteError> errorResult: minioClient.removeObject("my-bucketname", objectNames)) {
+   * for (Result<DeleteError> errorResult: minioClient.removeObjects("my-bucketname", objectNames)) {
    *     DeleteError error = errorResult.get();
    *     System.out.println("Failed to remove '" + error.objectName() + "'. Error:" + error.message());
    * } }</pre>
    *
    * @param bucketName Bucket name.
    * @param objectNames List of Object names in the bucket.
    */
-  public Iterable<Result<DeleteError>> removeObject(final String bucketName, final Iterable<String> objectNames) {
+  public Iterable<Result<DeleteError>> removeObjects(final String bucketName, final Iterable<String> objectNames) {
```

Comment:
All other SDKs, Go, JavaScript, Python, .NET, have 2 methods to remove objects:
* `removeObject()` for single object removal
* `removeObjects()` for multiple object removal

I think we need to stick with that approach for consistency  purposes.

Reply:
This is under discussion . @deekoder can you please confirm on this.

Reply:
By the way, no matter which way we choose to go about renaming the method, Bala's point about breaking compatibility is going to be still valid.